src/screens/OtherProfile/index.js

```diff
@@ -640,12 +640,9 @@ const styles = StyleSheet.create({
     marginLeft: 18
   },
   profileImage: {
-    width: 94,
-    height: 94,
-    borderRadius: 100,
-    // marginLeft: 3,
-    marginTop: 3,
-    marginBottom: 3
+    width: 95,
+    height: 95,
+    borderRadius: 100
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dimensions of the profile image have been slightly increased and the marginLeft, marginTop, and marginBottom properties have been removed. This could potentially affect the layout and positioning of the image within its parent component. Make sure these changes don't cause any undesired shifts in the UI.
```diff
-    width: 94,
-    height: 94,
-    borderRadius: 100,
-    // marginLeft: 3,
-    marginTop: 3,
-    marginBottom: 3
+    width: 95,
+    height: 95,
+    borderRadius: 100
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->